classes/PipelineRequest/mock.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Mocked PipelineRequest.
+ * Use it any time you need to mock a PipelineRequest.
+ *
+ * For more information and usage examples, please check the readme.
+ */
+class MockedPipelineRequest {
+  /**
+   * Getter for mockedDispatchResolver which is additional helper function for custom, mock-only
+   * .dispatch resolver.
```

Comment:
For clarity, when describing local functions use `dispatch()` instead of `.dispatch` since a period is used for punctuation.

Reply:
done